### DISCUSSION TITLE
CareTeam

### DISCUSSION THREAD

**Discussion by dt-r** [hl7au/au-fhir-core#12] [AU Core R1 Resource Priority List - CLOSED]
Upvote for inclusion of a Resource Profile of [CareTeam](http://hl7.org/fhir/R4/careteam.html) Resource in AU Core Release 1.

Comment if you think we need a different profile to AU Core CareTeam, or if there is a need for an additional CareTeam profile in AU Core.

AU Core CareTeam profile would set minimum expectations for a CareTeam resource to record, search, and fetch information about a care team. 

AU Core CareTeam should be based on the [AU Base CareTeam](http://build.fhir.org/ig/hl7au/au-fhir-base/StructureDefinition-au-careteam.html) profile and identify the additional mandatory core elements, extensions, vocabularies and value sets that SHALL be present in the CareTeam resource when conforming to this profile. It provides the floor for standards development for specific uses cases in an Australian context.

**Comment by jackieoc370** [comment]
If i'm understanding this aspect properly, this is critical for allied health professionals to more efficiently liaise with all the different stakeholders they need to for multi-disciplinary care - that is, identify the people involved, their contacts to be able to liaise and share treatment / care plans to enable implementation at lower cost to the health system, reduce duplication of referrals to others who need to be involved that they may not otherwise be aware of as consumer has not mentioned etc. This is particularly relevant in the disability sector
